src/Pangraph/AlgebraicGraphs.hs

```diff
@@ -0,0 +1,9 @@
+module Pangraph.AlgebraicGraphs
```

Comment:
There is no need for this module. Users can simply do `toGraph` to convert `Pangraph` values to any Alga graph, including `Graph a`.